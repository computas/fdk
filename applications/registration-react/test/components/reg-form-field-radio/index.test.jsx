import React from 'react';
import { shallow } from 'enzyme';
import RadioField from '../../../src/components/reg-form-field-radio';

test('should render RadioField correctly', () => {
  const wrapper = shallow(<RadioField />);
  expect(wrapper).toMatchSnapshot();
});
