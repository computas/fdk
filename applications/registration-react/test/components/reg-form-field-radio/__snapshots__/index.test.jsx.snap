// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render RadioField correctly 1`] = `
<label
  className="form-check fdk-form-check mb-0"
  htmlFor=""
>
  <input
    className="form-check-input"
    id=""
    type="radio"
  />
  <span
    className="form-check-label fdk-form-check-label"
  />
</label>
`;
